UPGRADE-7.0.md

```diff
@@ -107,3 +107,9 @@ The code entities in this list were marked as deprecated in previous releases an
 ## Database Tables
 
 ## Database Fields
+
+#Icons
+
+All icons in CoreBundle/Resources/public/images/ 
+and CoreBundle/Resources/public/themes/standard/images/ (not recursively, subfolders remain) where deleted. 
```

Comment:
"were deleted"

Also in the ticket/fix description.

Reply:
fixed.